.github/workflows/test-nightly.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v4
         with:
-          node-version: 21-nightly
+          node-version: 22-nightly
```

Comment:
Feels wrong to bump it manually. I was trying `nightly` and `*-nightly`, those do not work. Looking at action's code seems like number must be provided. Can be I misunderstood the code.

Reply:
yeah, same reason why I put 21 there originally